### DISCUSSION TITLE
Change number of epochs for Your_tts model in train_yourtts.py

### DISCUSSION THREAD

**Discussion by prachitui** [coqui-ai/TTS#2696] [General Q&A]
Hello, I am trying to train the your_tts model and it has been running since past 2.5 weeks. So, I thought I will just change the number of epochs for  train_yourtts.py file but I cannot find where can I make that change. If someone knows, help will be much appreciated. :)

**Comment by Bestom927** [comment]
Add 
epochs=1000,  
in
VitsConfig
It just uses the default epochs=1000, nondisplay
